custom/icds_reports/management/commands/imbalanced_indicator.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import absolute_import, print_function
+from __future__ import unicode_literals
+
+from django.core.management.base import BaseCommand
+
+import dateutil
+from corehq.sql_db.connections import get_icds_ucr_db_alias
+from django.db import connections
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            'input_month'
+        )
+
+    def handle(self, input_month, *args, **options):
+        indicator_list = self.get_indicator_list()
+        indicator_ratio = []
+
+        for indicator in indicator_list:
+            input_month = dateutil.parser.parse(input_month).date().replace(day=1).__str__()
+            query = self.prepare_query(indicator, input_month)
+            ratio_check = self.execute_query(query)
+            ratio_check = [(indicator["indicators"][i]['numerator'], res) for i, res in enumerate(ratio_check)]
+            indicator_ratio.extend(ratio_check)
+
+        for indicator in indicator_ratio:
+            print("{0}: {1}".format(*indicator))
+
+    def execute_query(self, query):
+        db_alias = get_icds_ucr_db_alias()
+        with connections[db_alias].cursor() as cursor:
+            cursor.execute(query)
+            return cursor.fetchall()[0]
+
+    def prepare_query(self, indicator_dict, input_month):
+        query_param = dict()
+        query_param['table_name'] = indicator_dict['table']
+        indicators = ""
+        for indicator in indicator_dict['indicators']:
+            indicators += "({numerator})>({denominator}),".format(**indicator)
```

Comment:
I don't think this will work. The last indicator can't have a trailing comma. It will also be hard to track what indicator these correspond to unless you add a column name `x > y as name` which should be easy to add to the dict below

Reply:
Both the problems are already captured:
1. about the trailing command on line : 45
2. about tracking corresponding column: on line 26

Reply:
so output of indicator becomes something like
indicator1(unique numerator): True/False
indicator2(unique numerator): True/False
indicator3(unique numerator): True/False

Reply:
already commented on the numerator part of that above, but this would be cleaner/easier to read if instead of constructing a giant string, you built a list and then used `join`, which would also handle the trailing comma. something like:
```
indicators = []
for indicator in indicator_dict['indicators']:
            indicators.append("({numerator})>({denominator})".format(**indicator)
columns = ', '.join(indicators)
```
and then you can use `columns` in the query below

Reply:
yeah, Nice. how I could not think of it. Disadvantage of night coding :D